new/LIT6723Yohanni.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6723Yohanni" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Martyrdom of Yoḥanni</title>
```

Comment:
I think this record is not needed and we should use https://betamasaheft.eu/works/LIT4051Homily/.

Reply:
LIT1521 and LIT4051 are on Yoḥanni of Dabra ʿAśā, of Tigray, surely not a martyr (Gadl YDA in BN Éth. 132, BL Add. 16,230, EMML nos. 1734, 3445, 4268, etc, this martyrdom is only in EMML 7602)
(I am not sure who BHO 496 is about... there may be an error in LIT1521)

Reply:
BHO 486 is also Y of DA, all correct